test/FilterTest.cpp

```diff
@@ -46,7 +47,10 @@ void checkSetPrefilterExpressionVariablePair(
     SparqlTripleSimple triple,
     std::unique_ptr<sparqlExpression::SparqlExpression> sparqlExpr,
     std::unique_ptr<prefilterExpressions::PrefilterExpression> prefilterExpr,
-    ColumnIndex columnIdx, bool prefilterIsApplicable) {
+    ColumnIndex columnIdx, bool prefilterIsApplicable,
+    bool enablePrefilterForFilter = true) {
+  RuntimeParameters().set<"enable-prefilter-on-index-scans">(
+      enablePrefilterForFilter);
```

Comment:
This should be done via the `setRuntimeParameterForTest` util from the `RuntimeParametersTestHelpers.h` file, s.t. the global state is restored in any case.